modules/modelops/pages/feast-teradata.adoc

```diff
@@ -0,0 +1,253 @@
+= Using Teradata with FEAST
+:experimental:
+:page-author: Mohammmad Taha Wahab , Mohammad Harris Mansur and Dr. Will Fleury
+:page-email: mohammadtaha.wahab@teradata.com , mohammadharris.mansur@teradata.com and will.fleury@teradata.com
+:page-revdate: January 5th, 2023
+:description: Feast Enterprise Feature Store Connector with Teradata
+:keywords: data warehouses, analytics, teradata, vantage, time series, business intelligence, enterprise analytics, feature store, Feast, connector
+
+== Introduction
+
+Feast's connector for Teradata is a complete implementation with support for all features and uses teradata as an online and offline store.
+
+== Prerequisites
+
+You need access to a Teradata Vantage instance.
+
+include::ROOT:partial$vantage.express.options.adoc[]
+
+
+== Overview
+We recommend you familiarize yourself with the terminology and concepts of feast by reading the official https://docs.feast.dev/[FEAST documentation]
```

Comment:
While the tone of all Getting Started articles is not consistent yet, we have decided to stick to an impersonal (but direct) tone. Instead of "we recommend", I would say "This how-to assumes you know Feast terminology. If you need a refresher check out ...."